stubs/Pillow/PIL/Image.pyi

```diff
@@ -65,13 +65,13 @@ class Transpose(IntEnum):
     TRANSVERSE = 6
 
 # All Transpose items
-FLIP_LEFT_RIGHT: Literal[0]
-FLIP_TOP_BOTTOM: Literal[1]
-ROTATE_90: Literal[2]
-ROTATE_180: Literal[3]
-ROTATE_270: Literal[4]
-TRANSPOSE: Literal[5]
-TRANSVERSE: Literal[6]
+FLIP_LEFT_RIGHT: Final = 0
```

Comment:
I tried to use the enum value here like this `FLIP_LEFT_RIGHT: Final = Transpose.FLIP_LEFT_RIGHT.value` but pytype complained.